### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#2567] [Announcements]
### Windows App SDK 1.1.0 is now available 🎉 

You are now able to use the stable version of Windows App SDK 1.1 in your applications! This release includes **new features *and* stability & performance improvements** across the component areas. New capabilities for apps using Windows App SDK include: 

- Apps can create **multiple windows** with WinUI3 content on the same UI thread. 
- **Mica** & **Background Acrylic** styles are available for WinUI 3 content.   
- **Improved default and custom title bar** with new functionalities like z-order, sizing/resizing, tall title bar, and extended show. 
- Apps using the Windows App SDK can **run with elevated permissions**. 
- Apps that deploy with all their dependencies together are now able to include the Windows App SDK as well.  **(self-contained deployment)** 
- Apps can **push notifications** regardless of being packaged/unpackaged, or whether the app is published in the Store. 
- The **environment variables** PATH, and PATHEXT can be managed and tracked at runtime using new convenient APIs.   
- Apps can **restart** manually or register themselves to be restarted if it was running when a system update occurs. 

**To see everything that's new and changed, see the full [Windows App SDK 1.1 release notes](https://aka.ms/windowsappsdk/1.1/1.1.0/release-notes).**  

### Getting Started 

To get started using Windows App SDK to develop Windows apps, check out the following documentation:    
- [Install developer tools ](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment).     
- [Create a WinUI 3 app](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app).   
- [Continue your development journey](https://docs.microsoft.com/windows/apps/develop).   

### What's next  

To keep up with progress being made on the Windows App SDK, please see our [feature roadmap](https://aka.ms/winappsdkroadmap). 

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by harvinders** [comment]
@gabbybilka In the release note, the link for c# mica is broken, https://github.com/microsoft/WinUI-Gallery/tree/winui3/XamlControlsGallery/ControlPagesSampleCode/SystemBackdrops

**Reply by shelllet** [reply]
please fix these two bugs, my app crashed upgrade to 1.1.

https://github.com/microsoft/microsoft-ui-xaml/issues/6999
https://github.com/microsoft/microsoft-ui-xaml/issues/7191

**Reply by otomad** [reply]
The new link is https://github.com/microsoft/WinUI-Gallery/tree/main/WinUIGallery/ControlPagesSampleCode/SystemBackdrops
